src/plone/restapi/tests/test_services_querystring.py

```diff
@@ -64,7 +64,14 @@ def test_endpoint_shows_field_config(self):
             "values": {},
             "vocabulary": None,
         }
-        self.assertEqual(expected_field_config, idx)
+        # Be permissive with the check and only check the existing
```

Comment:
@reebalazs this is certainly a smart solution. Though, when it comes to testing I prefer simplicity and readability over DRY and being smart. One of the most important assets of a test is that it explicitly explains what values we expect and that it returns meaningful responses to test failures (and does not require checking a loop).

I know this is a lot more to type but what I usually do is just add lots of those lines:

```
self.assertEqual("Text search of an item's title", idx.get("description"))
```

This approach is robust and returns an explicit failure when a value is missing.

The other option is to rely on "self.assertDictContainsSubset(".

As said, that's a matter of personal preference...